app/controllers/dashboards_controller.rb

```diff
@@ -1,13 +1,13 @@
+
```

Comment:
I'm really happy with the overall shape of this controller now.  Thoughts @jmejia?